sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -399,300 +537,380 @@
+          "overlay": {
+            "style": "",
+            "page": "",
+            "left": "",
+            "top": "",
+            "width": "",
+            "height": ""
+          },
+          "type": "htmlelement",
+          "input": false,
+          "tableView": false,
+          "placeholder": "",
+          "prefix": "",
+          "suffix": "",
+          "multiple": false,
+          "defaultValue": null,
+          "protected": false,
+          "unique": false,
+          "persistent": false,
+          "clearOnHide": true,
+          "refreshOn": "",
+          "redrawOn": "",
+          "dataGridLabel": false,
+          "labelPosition": "top",
+          "description": "",
+          "errorLabel": "",
+          "tooltip": "",
+          "hideLabel": false,
+          "tabindex": "",
+          "disabled": false,
+          "autofocus": false,
+          "dbIndex": false,
+          "customDefaultValue": "",
+          "calculateValue": "",
+          "calculateServer": false,
+          "widget": null,
+          "validateOn": "change",
+          "validate": {
+            "required": false,
+            "custom": "",
+            "customPrivate": false,
+            "strictDateValidation": false,
+            "multiple": false,
+            "unique": false
+          },
+          "allowCalculateOverride": false,
+          "encrypted": false,
+          "showCharCount": false,
+          "showWordCount": false,
+          "allowMultipleMasks": false,
+          "id": "ev5n9qc"
+        },
+        {
+          "label": "How will this be offered?",
+          "labelPosition": "top",
+          "optionsLabelPosition": "right",
+          "description": "",
+          "tooltip": "",
+          "customClass": "",
+          "tabindex": "",
+          "inline": false,
+          "hidden": false,
+          "hideLabel": false,
+          "autofocus": false,
+          "disabled": false,
+          "tableView": false,
+          "modalEdit": false,
+          "values": [
+            {
+              "label": "Part time",
+              "value": "Part Time",
+              "shortcut": ""
+            },
+            {
+              "label": "Full time",
+              "value": "Full Time",
+              "shortcut": ""
             }
           ],
-          "autoAdjust": false,
+          "dataType": "",
+          "persistent": true,
+          "protected": false,
+          "dbIndex": false,
+          "encrypted": false,
+          "redrawOn": "",
+          "clearOnHide": true,
+          "customDefaultValue": "",
+          "calculateValue": "",
+          "calculateServer": false,
+          "allowCalculateOverride": false,
+          "validate": {
+            "required": true,
+            "onlyAvailableItems": true,
+            "customMessage": "",
+            "custom": "valid=(data.programIntensity.includes(input))?true:'The program you have selected has only ' +data.programIntensity+ ' offering';",

```

Comment:
can you add proper spacing for better code readability

Reply:
It wasn't part of my change. But added spaces for better readability.